subprojects/docs/src/samples/userguide/dependencies/unresolvedTransitiveDependencies/build.gradle

```diff
@@ -0,0 +1,11 @@
+//START SNIPPET unresolved-transitive-dependencies
+apply plugin: 'java-library'
```

Comment:
Nitpick: `apply plugin: 'java'` instead

Otherwise it would be an example with bad practice. A library should never depend on a logging framework implementation. Only on it's API.

Reply:
You are right. Good comment. So we'd use the `java` plugin + the `compile` configuration here right?

Reply:
Changed it accordingly.

Reply:
No it should still use `implementation`. That is also available in the `java` plugin. basically `implementation` is the _new_ compile. But we did not call it compile, to not break the existing compile. 

I noticed however, that we did not make the change in the `java` plugin docs. We need to adjust it there as well. I added a comment here: https://github.com/gradle/dotorg-docs/issues/42#issuecomment-356232878

Reply:
done